ros_speech_recognition/test/sample_ros_speech_recognition.test

```diff
@@ -9,10 +13,12 @@
   </rosparam>
   <include file="$(find ros_speech_recognition)/launch/speech_recognition.launch">
```

Comment:
How about integrating the content of this test to `sample_ros_speech_recognition.launch`?
or
How about moving the content of this test to `sample_ros_speech_recognition_ja.launch`?